src/components/fb/CreateFidelityBond.tsx

```diff
@@ -617,7 +617,7 @@ const CreateFidelityBond = ({ otherFidelityBondExists, wallet, walletInfo, onDon
   }
 
   return (
-    <div className={styles.container}>
```

Comment:
On first glance, it seems the design has slightly changed, since the `container` css class moved away from here.

#### Before
<img src="https://github.com/joinmarket-webui/jam/assets/3358649/d1dba399-396a-451f-96c0-fb49091fa6b9" width=321 />

#### After

<img src="https://github.com/joinmarket-webui/jam/assets/3358649/a3b2adca-0d56-4624-a7e2-82f78a8439a1" width=321 />

Dropdown content is not within the surrounding area border anymore (no FB present).

Reply:
yes the code changes will make this issue. I kinda knew it. But If I try to fix then _<rb.collapse>_ section will have to be repeated twice for both the cases ( active bond exists and no active bond). This will just increase the code length.
One solution is that eventually it will be a modal so this issue shouldn't be there after we do that. So we can possibly fix that issue(modal instead of dropdown for the form) first and then there will be nothing to talk about this.